### ISSUE TITLE
i retrain centernet,and i got the loss is Nan

### ISSUE THREAD

**Issue by LCWdmlearning** [Stick-To/Object-Detection-API-Tensorflow#6]
i retrain centernet on new dataset,when i training to about 10 epochs,i got loss is Nan.and then ,
I training at the ninth epoch and loss returned to normal.but when I train two more epochs,
the loss become Nan again.can you help me?

**Comment by Stick-To** [comment]
maybe the learning_rate is not appropriate

**Comment by LCWdmlearning** [comment]
> maybe the learning_rate is not appropriate

Thank you for your reply. i set the learning_rate to 0.001，batchsize is 8.
i try to change learning_rate to 0.0001 and i got the same problem when i  I train two more epochs.

**Comment by Stick-To** [comment]
batch size is too small, it can't make batch_normalization helpful
